src/transaction/transaction_master_group_complete_manager.cpp

```diff
@@ -149,8 +171,8 @@ namespace cubtx
   //
   void master_group_complete_manager::do_complete (THREAD_ENTRY *thread_p)
   {
-    LOG_LSA closed_group_commit_lsa;
-    LOG_TDES *tdes = logtb_get_tdes (&cubthread::get_entry ());
+    LOG_LSA closed_group_start_complete_lsa, closed_group_end_complete_lsa;
```

Comment:
I think the names of groups should be simplified.
lastest_closed_group  -> why don't we just refer to it as 'prev_group'; since prev group is closed (we can't add more transaction to it)

is_latest_closed_group_completed   -> is_prev_group_completed
is_latest_closed_group_prepared_for_complete  -> is_prev_group_prepared_for_complete
starts_latest_closed_group_complete  -> set_prev_group_complete_started 
get_latest_closed_group -> get_prev_group

Reply:
Renaming and other code clean will be considered at CBRD-22947,  in order to be done only once.